internal/clients/team5/iito.go

```diff
@@ -101,10 +116,13 @@ func (c *client) GetGiftResponses(receivedOffers shared.GiftOfferDict) shared.Gi
 			AcceptedAmount: shared.Resources(offer), // Accept all they gave us
 			Reason:         shared.Accept,           // Accept all gifts duh
 		}
-		newGiftRequest := giftInfo{ // 	For each client create a new gift info
-			requested: c.giftHistory[team].OurRequest[c.gameState().Turn].requested, // Amount requested (from above)
-			gifted:    shared.GiftOffer(responses[team].AcceptedAmount),             // Amount accepted
-			reason:    shared.AcceptReason(responses[team].Reason),                  // Reason accepted
+	}
+	// History
+	for team := range c.gameState().ClientLifeStatuses { // Could be in the loop above but
+		newGiftRequest := giftInfo{ // for consistency with other functions its here
+			requested: c.giftHistory[team].OurRequest[c.gameState().Turn].requested, // Amount We requested
+			offered:   receivedOffers[team],                                         // Amount offered TO US
+			response:  responses[team],                                              // Amount and reason WE accepted
 		}
 		c.giftHistory[team].OurRequest[c.gameState().Turn] = newGiftRequest
```

Comment:
Should `OurRequest` and `TheirRequest` be exported? If you've defined them in our package, they shouldn't be.